docs/ci/demo

```diff
@@ -249,9 +252,10 @@ set +e
 init_test
 popper search quiho
 popper search linux --skip-update
-test $? - eq 0
+test $? -eq 0
```

Comment:
when `set -e` is specified, then any non-zero exit code will cause the script to exit. So, in this case, if we remove line 251 (where `set +e` is specified) this line is not needed, as well as the following one (256)